geode-core/src/main/java/org/apache/geode/distributed/AbstractLauncher.java

```diff
@@ -156,7 +166,14 @@ protected static Properties loadGemFireProperties(final URL url) {
 
     if (url != null) {
       try {
-        properties.load(new FileReader(new File(url.toURI())));
+        File propertyFile = new File(url.toURI());
```

Comment:
Updated.  testLoadGemFirePropertiesWithNonExistingURL is an existing test that exercises this code.